packages/domain-picker/src/domain-picker/style.scss

```
@@ -374,6 +374,14 @@ $accent-blue: #117ac9;
 	}
 }
 
+.domain-picker__suggestion-item.is-free {
+	// free domains don't include the wide phrase "Included in paid plans"
+	// they only include "Free" or "Default" word, this shrinks their width 
+	.domain-picker__price {
+		flex-grow: 0.1 // this is practically the same as 0. But 0 breaks IE.
```

Like @razvanpapadopol suggested, I would go for `flex-grow: 1`, like they do it for the P2 todo-list checkbox items (see: pbAok1-1PR-p2).
```suggestion
		flex-grow: 1;
```

---

@StefanNieuwenhuis in that example they are applying the `flex-grow: 1` to the item they want to occupy the full space which in our case is `domain-picker__suggestion-item-name` and not the price.

---

Exactly. The item that we need to remain wide already has `flex: 1`.

---

Sorry for that! When I woke up, I realized this too...